elephant/spike_train_processing.py

```diff
@@ -0,0 +1,317 @@
+# -*- coding: utf-8 -*-
+"""
+Module for spike train processing
+
+:copyright: Copyright 2014-2020 by the Elephant team, see `doc/authors.rst`.
+:license: Modified BSD, see LICENSE.txt for details.
+"""
+
+from __future__ import division
+
+import neo
+import elephant.conversion as conv
+import numpy as np
+from .utils import _check_consistency_of_spiketrainlist
+
+
+def _get_index(lst, obj):
+    for index, item in enumerate(lst):
+        if item is obj:
+            return index
+
+
+def detect_synchrofacts(spiketrains, sampling_rate, spread=1,
+                        deletion_threshold=None, invert_delete=False):
+    """
+    Given a list of neo.Spiketrain objects, calculate the number of synchronous
+    spikes found and optionally delete or extract them from the given list
+    *in-place*.
```

Comment:
Should be "a list of `neo.Spiketrain` objects"

Reply:
You're definitely giving Alex a hard time!
:D
I didn't dare to comment on such things. I think they are negligible anyhow, especially since the API is not yet confirmed.

Reply:
:D. Just to flag things in advance. Should the API change, the documentation may just be moved around.